app/scripts/modules/core/src/managed/ArtifactDetail.tsx

```diff
@@ -230,7 +236,34 @@ export const ArtifactDetail = ({
               Mark as bad...
             </Button>
           </div>
-          <div className="detail-section-right">{/* artifact metadata will live here */}</div>
+          <div className="detail-section-right flex-container-v flex-pull-right sp-margin-l-right">
+            {git?.author && <VersionMetadataItem label="Author" value={git.author} />}
+            {git?.pullRequest.number && git.pullRequest.url && (
```

Comment:
what will happen if `git.pullRequest` will be null?

Reply:
(like in a master build)

Reply:
I was going to mention this as some API feedback — it seems that when a version is not associated with a PR the `pullRequest` object itself still exists, with _both_ a `number` and `url` field, but each of those fields are empty strings. This check gets around that by testing both fields for truthiness (which empty strings won't pass), but ideally I'd like it if the API just left out the `pullRequest` object entirely when there's no PR.

If you are willing to do that on the API side, I will update this with a null check like you suggest so that the API change doesn't break this line

Reply:
updated the checks here to be tolerant to `pullRequest` being null.

Reply:
Makes sense - let me see how to omit `pullRequest` in case it's null.